uftrace.h

```diff
@@ -72,6 +72,7 @@ enum uftrace_feat_bits {
 };
 
 enum uftrace_info_bits {
+	UFTRACE_VER,
```

Comment:
Please make it simply `VERSION` unless there's a name conflict.  Or maybe we could rename them all to have `INFO_` prefix.